### ISSUE TITLE
Editor crashes when importing meshes without a material

### ISSUE THREAD

**Issue by antourenein** [godotengine/godot#61665]
### Godot version

4.0 alpha 9

### System information

Windows 10, Vulkan, NVIDIA GTX 850M

### Issue description

Importing a GLTF scene containing a mesh without a material consistently crashes the editor when clicking the material in the Material tab or trying to extract materials.

### Steps to reproduce

Bring the attached GLB to the file system (it is just a simple scene with a box mesh and no material assigned to it, exported from Blender 3.1), double click it to open the advanced import menu, go to the Material tab and click the empty material slot (or go to Actions... > Extract Materials and choose any folder)

The editor should crash.

### Minimal reproduction project

GLB Scene: [box.zip](https://github.com/godotengine/godot/files/8834482/box.zip)

**Comment by qarmin** [comment]
```
editor/import/scene_import_settings.cpp:722:34: runtime error: member call on null pointer of type 'struct TreeItem'
editor/import/scene_import_settings.cpp:722:34: runtime error: member access within null pointer of type 'struct TreeItem'

================================================================
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.0.alpha.custom_build (1baee2189c21ae36c852ddff10f769b1134509ca)
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] godot4s() [0x243f6d8] (/home/runner/work/GodotBuilds/GodotBuilds/godot/platform/linuxbsd/crash_handler_linuxbsd.cpp:56)
[2] /lib/x86_64-linux-gnu/libc.so.6(+0x42520) [0x7fe0753d1520] (??:0)
[3] SceneImportSettings::_select(Tree*, String, String) (/home/runner/work/GodotBuilds/GodotBuilds/godot/editor/import/scene_import_settings.cpp:722)
[4] SceneImportSettings::_material_tree_selected() (/home/runner/work/GodotBuilds/GodotBuilds/godot/editor/import/scene_import_settings.cpp:788 (discriminator 5))
[5] void call_with_variant_args_helper<SceneImportSettings>(SceneImportSettings*, void (SceneImportSettings::*)(), Variant const**, Callable::CallError&, IndexSequence<>) (/home/runner/work/GodotBuilds/GodotBuilds/godot/./core/variant/binder_common.h:241 (discriminator 4))
[6] void call_with_variant_args<SceneImportSettings>(SceneImportSettings*, void (SceneImportSettings::*)(), Variant const**, int, Callable::CallError&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/./core/variant/binder_common.h:336)
[7] CallableCustomMethodPointer<SceneImportSettings>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/runner/work/GodotBuilds/GodotBuilds/godot/./core/object/callable_method_pointer.h:104)
[8] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/runner/work/GodotBuilds/GodotBuilds/godot/core/variant/callable.cpp:50)
[9] Object::emit_signalp(StringName const&, Variant const**, int) (/home/runner/work/GodotBuilds/GodotBuilds/godot/core/object/object.cpp:1120)
[10] Error Object::emit_signal<>(StringName const&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/./core/object/object.h:773)
[11] Tree::select_single_item(TreeItem*, TreeItem*, int, TreeItem*, bool*, bool) (/home/runner/work/GodotBuilds/GodotBuilds/godot/scene/gui/tree.cpp:2351)
[12] Tree::select_single_item(TreeItem*, TreeItem*, int, TreeItem*, bool*, bool) (/home/runner/work/GodotBuilds/GodotBuilds/godot/scene/gui/tree.cpp:2388 (discriminator 6))
[13] Tree::propagate_mouse_event(Vector2i const&, int, int, int, bool, TreeItem*, MouseButton, Ref<InputEventWithModifiers> const&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/scene/gui/tree.cpp:2609)
[14] Tree::propagate_mouse_event(Vector2i const&, int, int, int, bool, TreeItem*, MouseButton, Ref<InputEventWithModifiers> const&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/scene/gui/tree.cpp:2771)
[15] Tree::gui_input(Ref<InputEvent> const&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/scene/gui/tree.cpp:3472 (discriminator 10))
[16] Control::_call_gui_input(Ref<InputEvent> const&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/scene/gui/control.cpp:939)
[17] Viewport::_gui_call_input(Control*, Ref<InputEvent> const&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/scene/main/viewport.cpp:1302)
[18] Viewport::_gui_input_event(Ref<InputEvent>) (/home/runner/work/GodotBuilds/GodotBuilds/godot/scene/main/viewport.cpp:1537 (discriminator 1))
[19] Viewport::push_input(Ref<InputEvent> const&, bool) (/home/runner/work/GodotBuilds/GodotBuilds/godot/scene/main/viewport.cpp:2741 (discriminator 3))
[20] Window::_window_input(Ref<InputEvent> const&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/scene/main/window.cpp:974 (discriminator 2))
[21] void call_with_variant_args_helper<Window, Ref<InputEvent> const&, 0ul>(Window*, void (Window::*)(Ref<InputEvent> const&), Variant const**, Callable::CallError&, IndexSequence<0ul>) (/home/runner/work/GodotBuilds/GodotBuilds/godot/./core/variant/binder_common.h:236 (discriminator 6))
[22] void call_with_variant_args<Window, Ref<InputEvent> const&>(Window*, void (Window::*)(Ref<InputEvent> const&), Variant const**, int, Callable::CallError&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/./core/variant/binder_common.h:336)
[23] CallableCustomMethodPointer<Window, Ref<InputEvent> const&>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/runner/work/GodotBuilds/GodotBuilds/godot/./core/object/callable_method_pointer.h:104)
[24] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/runner/work/GodotBuilds/GodotBuilds/godot/core/variant/callable.cpp:50)
[25] DisplayServerX11::_dispatch_input_event(Ref<InputEvent> const&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/platform/linuxbsd/display_server_x11.cpp:3134)
[26] DisplayServerX11::_dispatch_input_events(Ref<InputEvent> const&) (/home/runner/work/GodotBuilds/GodotBuilds/godot/platform/linuxbsd/display_server_x11.cpp:3108 (discriminator 4))
[27] Input::_parse_input_event_impl(Ref<InputEvent> const&, bool) (/home/runner/work/GodotBuilds/GodotBuilds/godot/core/input/input.cpp:631)
[28] Input::flush_buffered_events() (/home/runner/work/GodotBuilds/GodotBuilds/godot/core/input/input.cpp:884)
[29] DisplayServerX11::process_events() (/home/runner/work/GodotBuilds/GodotBuilds/godot/platform/linuxbsd/display_server_x11.cpp:3400)
[30] OS_LinuxBSD::run() (/home/runner/work/GodotBuilds/GodotBuilds/godot/platform/linuxbsd/os_linuxbsd.cpp:439)
[31] godot4s(main+0x496) [0x243cd7c] (/home/runner/work/GodotBuilds/GodotBuilds/godot/platform/linuxbsd/godot_linuxbsd.cpp:70)
[32] /lib/x86_64-linux-gnu/libc.so.6(+0x29d90) [0x7fe0753b8d90] (??:0)
[33] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x80) [0x7fe0753b8e40] (??:0)
[34] godot4s(_start+0x2e) [0x243c82e] (??:?)
-- END OF BACKTRACE --
================================================================
```

**Comment by fire** [comment]
Probably fixed by https://github.com/godotengine/godot/pull/61678. Same function.

**Comment by akien-mga** [comment]
Fixed by #59834.
